lib/Controller/ViewController.php

```diff
@@ -100,6 +100,7 @@ public function index():TemplateResponse {
 		$showWeekNumbers = $this->config->getUserValue($this->userId, $this->appName, 'showWeekNr', $defaultWeekNumbers) === 'yes';
 		$skipPopover = $this->config->getUserValue($this->userId, $this->appName, 'skipPopover', $defaultSkipPopover) === 'yes';
 		$timezone = $this->config->getUserValue($this->userId, $this->appName, 'timezone', $defaultTimezone);
+		$attachmentsFolder = $this->config->getUserValue($this->userId, $this->appName, 'attachmentsFolder', '/');
```

Comment:
Save as above, use `dav` instead of `calendar`

Reply:
https://github.com/nextcloud/calendar/pull/4251/commits/5a5638c6ff7a1b7d336b1fc8a90b913a8d32bb8b